doc/src/manual/parallel-computing.md

```diff
@@ -272,8 +282,8 @@ remotecall_fetch(()->foo(A), 3)  # worker 3
 A = nothing
 ```
 
-  Executing the above snippet results in `Main.A` on worker 2 having a different value from
-  `Main.A` on worker 3, while the value of `Main.A` on node 1 is set to `nothing`.
+Executing the above snippet results in `Main.A` on worker 2 having a different value from
+`Main.A` on worker 3, while the value of `Main.A` on node 1 is set to `nothing`.
```

Comment:
Alright, it didn't render as such so I removed the spaces. perhaps the code block needs to be indented too, to not break the bullet list?